pages/api/sourcecred.js

```diff
@@ -0,0 +1,12 @@
+import { getSourcecredContributions } from '../../util/sourcecred'
+
+export default async (req, res) => {
```

Comment:
Simple lint warning here and elsewhere: just assign this function to a const and then export default [constName]...